pkg/apis/core/validation/validation_test.go

```diff
@@ -902,41 +881,17 @@ func TestValidatePersistentVolumeSourceUpdate(t *testing.T) {
 
 func TestValidationOptionsForPersistentVolume(t *testing.T) {
 	tests := map[string]struct {
-		oldPv                  *core.PersistentVolume
-		enableReadWriteOncePod bool
-		expectValidationOpts   PersistentVolumeSpecValidationOptions
+		oldPv                *core.PersistentVolume
+		expectValidationOpts PersistentVolumeSpecValidationOptions
 	}{
 		"nil old pv": {
-			oldPv:                  nil,
-			enableReadWriteOncePod: true,
+			oldPv: nil,
 			expectValidationOpts: PersistentVolumeSpecValidationOptions{
 				AllowReadWriteOncePod: true,
 			},
 		},
-		"rwop allowed because feature enabled": {
-			oldPv:                  pvWithAccessModes([]core.PersistentVolumeAccessMode{core.ReadWriteOnce}),
-			enableReadWriteOncePod: true,
-			expectValidationOpts: PersistentVolumeSpecValidationOptions{
-				AllowReadWriteOncePod: true,
-			},
-		},
-		"rwop not allowed because not used and feature disabled": {
-			oldPv:                  pvWithAccessModes([]core.PersistentVolumeAccessMode{core.ReadWriteOnce}),
-			enableReadWriteOncePod: false,
-			expectValidationOpts: PersistentVolumeSpecValidationOptions{
-				AllowReadWriteOncePod: false,
-			},
-		},
-		"rwop allowed because used and feature enabled": {
```

Comment:
Should we keep the test cases where oldPV uses RWOP? Ditto throughout

Reply:
The new tests "rwop allowed" do this. I re-added the tests that check if RWOP is allowed when using a non-RWOP (RWO) access mode just to be explicit that the access mode is allowed in all cases.

Reply:
Update -- removed these tests since we're removing feature-gating logic. The method under test returns an empty struct so there's only one test to cover that case.